### PR TITLE
Add ignore_older to remaining nginx and system data streams

### PR DESCRIPTION
**Pull request by fearful-symmetry**
## What does this PR do?

This closes https://github.com/elastic/integrations/issues/3147 and adds `ignore_older` fields for the remaining data streams in `system`, as well as `nginx`.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Review by belimawr — APPROVED** [review]

**Comment by cmacknz** [comment]
Is `ignore_older` already available for the Windows integrations listed here? https://github.com/elastic/integrations/issues/3148

If not can you add those as well?

File: packages/nginx/data_stream/access/manifest.yml [hunk 1]
```diff
@@ -36,6 +36,14 @@ streams:
         description: >
           Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the logs are parsed. See [Processors](https://www.elastic.co/guide/en/beats/filebeat/current/filtering-and-enhancing-data.html) for details.
 
+      - name: ignore_older
+        type: text
+        title: Ignore events older than
+        default: 72h
```
Comment (ManojS-shetty) [inline packages/nginx/data_stream/access/manifest.yml:42]: Any specific reason we are having 72h as a default?

**Review by ManojS-shetty — APPROVED** [review]

File: packages/nginx/data_stream/access/manifest.yml (hunk 1 above)
Comment (fearful-symmetry) [reply packages/nginx/data_stream/access/manifest.yml:42]: It looks like at least a few other data streams set the same default, so I went with that.

File: packages/nginx/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.5.0"
+  changes:
+    - description: Add ignore_older to log data streams
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/3690
```
Comment (DJRickyB) [inline packages/nginx/changelog.yml:6]: drive-by, because I happened to be looking, this is #3691
